identity-service/src/routes/healthCheck.js

```diff
@@ -488,8 +497,14 @@ module.exports = function (app) {
           isWithinBounds(notificationJobLastSuccess) &&
           isWithinBounds(notificationEmailsJobLastSuccess) &&
           isWithinBounds(notificationAnnouncementsJobLastSuccess)
+        req.logger.info(
+          `notifications_check | isWithinBounds is ${isWithinBounds} and notifBlockDiff is ${notifBlockDiff}`
```

Comment:
you're printing out a method here. think you wanted `withinBounds`

Reply:
oo great catch thank you!